tests/test_alerts.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+
+
+@pytest.fixture
+def browser():
+    driver = webdriver.Chrome()
+    # driver = webdriver.Firefox()
+    driver.implicitly_wait(5)
+    driver.get("https://demo.seleniumeasy.com/javascript-alert-box-demo.html")
+    yield driver
+    driver.quit()
+
+
+def test_java_script_alert_box_text(browser):
+    click_me_button = browser.find_element(By.CSS_SELECTOR, "div.col-md-6.text-left div.panel.panel-primary:first-of-type button")
+    click_me_button.click()
+    # Verify text in opened alert
+    WebDriverWait(browser, 5).until(ec.alert_is_present())
+    java_script_alert_box = browser.switch_to.alert
+    actual_java_script_alert_box_text = java_script_alert_box.text
+    expected_java_script_alert_box_text = 'I am an alert box!'
+    assert actual_java_script_alert_box_text in expected_java_script_alert_box_text, f"Expected {expected_java_script_alert_box_text} to be presented, but got: {actual_java_script_alert_box_text}"
+    java_script_alert_box.accept()
```

Comment:
you don't need to do actions after assert

Reply:
it is done